.gitattributes

```diff
@@ -1,2 +1,5 @@
 *.jar filter=lfs diff=lfs merge=lfs -text
 *.ico filter=lfs diff=lfs merge=lfs -text
+src/build/docker/entrypoint.sh -text
```

Comment:
This file gets messed up by CRLF conversion, causing the docker container to break